### PR TITLE
feat: add type/epics label to Cloud repos

### PR DESCRIPTION
**Pull request by martzoukos**
This is for introducing the `type/epic` label to all of the Cloud related projects.

### CODE REVIEW

File: src/cloud/labels.ts [hunk 1]
```diff
@@ -166,112 +166,113 @@ export const groups = {
    * Useful labels for most things.
    */
   common: [
```
Comment (jasonkuhrt) [inline src/cloud/labels.ts:168]: I guess we can give up on the attempt at pretty whitespace in this file :)

**Review by jasonkuhrt — APPROVED** [review]
lgtm

**Comment by jasonkuhrt** [comment]
@janpio weird I seem to be able to push to master of this repo but not merge PRs? 🙃

File: src/cloud/labels.ts (hunk 1 above)
Comment (janpio) [reply src/cloud/labels.ts:168]: You could always block PRs that do so 🐌

**Pull request merged**